lib/segment/src/fixtures/payload_context_fixture.rs

```diff
@@ -165,7 +166,7 @@ impl IdTracker for FixtureIdTracker {
 ///
 /// Payload storage fixture
 ///
-fn create_payload_storage_fixture(num_points: usize, seed: u64) -> InMemoryPayloadStorage {
+pub fn create_payload_storage_fixture(num_points: usize, seed: u64) -> InMemoryPayloadStorage {
```

Comment:
Because it is a fixture I guess we want to limit the scope. If it can't be private, maybe we can do `pub(super)` or `pub(crate)` (didn't evaluate which one is required).

Reply:
We are using fixtures in the `benches` folder, to which rust is considered outside of the crate. This is why most of the items in the `fixtures` mod is public too, `pub(crate)` is not enough in this case, unless we move the whole module outside of the `segment` crate.